teams/team-geordie/dev/templates/yelb-app.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: yelb
+  namespace: argocd
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  destination:
+    namespace: geordie
```

Comment:
does it have to be in geordi namespace? So far we only created apps in argocd namespace, due to restrictions.

Reply:
Yes it is `geordie` namespace under spec. Please other sample teams. Application will stay in `argocd` but the the deployment will happen in `geordie` namespace which is our namespace for observability.

Reply:
nm, unsure where I was looking, not enough coffee.